### DISCUSSION TITLE
Change page navigation title "on this page".

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#3783] [Q&A]
How can I change the page navigation title "on this page" ? (ex in  "sommaire").

**Comment by mcanouil** [comment]
As any text component in Quarto you can change "translation":

```
language:
  toc-title-website: "A random title for TOC"
```

See <https://quarto.org/docs/authoring/language.html>.

**Comment by wilcar** [comment]
I am creating a website. 
I downloaded the french yml file on github. 
I called it "language.yml". 
But when I want to add  language: language.yml to the  _quarto.yml, language is not known as a property.

    project:
       type: website

    website:
     language: language.yml
      title: "Manuel"
      page-navigation: true

**Reply by mcanouil** [reply]
You only have to overwrite the not desired translation as described in the documentation, and in your case using only the two lines I provided in your file/project yaml.
Simply follow the documentation.
